src/RootedTrees.jl

```diff
@@ -4,6 +4,8 @@ module RootedTrees
 
 using LinearAlgebra: dot
 
+using LaTeXStrings
```

Comment:
Could you please specify which parts of `LaTeXStrings` you want to use explicitly as done for the other packages below? This clarifies the origin of stuff and avoids possible name clashes if packages `export` new stuff in the future.